server/start.sh

```diff
@@ -17,4 +17,6 @@ read_file_and_export "DB_USERNAME_FILE" "DB_USERNAME"
 read_file_and_export "DB_PASSWORD_FILE" "DB_PASSWORD"
 read_file_and_export "REDIS_PASSWORD_FILE" "REDIS_PASSWORD"
 
+export UV_THREADPOOL_SIZE=$CPU_CORES
```

Comment:
You wouldn't need an export if you just set UV_THREADPOOL_SIZE directly in the .env file.

Reply:
This is a draft PR